public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -235,8 +235,8 @@ export default class LokiLanguageProvider extends LanguageProvider {
   ): Promise<TypeaheadOutput> {
     let context = 'context-labels';
     const suggestions: CompletionItemGroup[] = [];
-    const line = value.anchorBlock.getText();
-    const cursorOffset = value.selection.anchor.offset;
+    const line = value?.anchorBlock.getText();
+    const cursorOffset = value?.selection.anchor.offset;
```

Comment:
Doing a null check two times on the same variable should be a code smell here,  adding if (!value) return earlier in the function could help